tests/python/driver/tvmc/test_frontends.py

```diff
@@ -211,3 +212,43 @@ def test_load_model___wrong_language__to_pytorch(tflite_mobilenet_v1_1_quant):
             model_format="pytorch",
             shape_dict={"input": [1, 3, 224, 224]},
         )
+
+
+@mock.patch("tvm.driver.tvmc.frontends.import_keras")
+def test_import_keras_friendly_message(import_keras_mock, keras_resnet50):
```

Comment:
```suggestion
def test_import_keras_friendly_message(import_keras_mock, keras_resnet50):
    # some CI environments wont offer keras, so skip in case it is not present
    pytest.importorskip("keras")
```

Reply:
Shouldn't this test be immune to whether or not `keras` is present?

Reply:
agreed--unless there's some strange impl detail here? i'd prefer if we could not skip these based on whether those packages are present.

Reply:
Yes, I have removed these lines as in this case it seems that they are not needed